2024-03-06 09:46:20,545 INFO    StreamThr :16016 [internal.py:wandb_internal():86] W&B internal server running at pid: 16016, started at: 2024-03-06 09:46:20.544677
2024-03-06 09:46:20,546 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status
2024-03-06 09:46:20,548 INFO    WriterThread:16016 [datastore.py:open_for_write():87] open: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\run-oyjusw69.wandb
2024-03-06 09:46:20,548 DEBUG   SenderThread:16016 [sender.py:send():378] send: header
2024-03-06 09:46:20,548 DEBUG   SenderThread:16016 [sender.py:send():378] send: run
2024-03-06 09:46:22,679 INFO    SenderThread:16016 [dir_watcher.py:__init__():211] watching files in: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files
2024-03-06 09:46:22,679 INFO    SenderThread:16016 [sender.py:_start_run_threads():1099] run started: oyjusw69 with start time 1709698580.018209
2024-03-06 09:46:22,813 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: check_version
2024-03-06 09:46:22,813 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: check_version
2024-03-06 09:46:23,124 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: run_start
2024-03-06 09:46:23,133 DEBUG   HandlerThread:16016 [system_info.py:__init__():26] System info init
2024-03-06 09:46:23,133 DEBUG   HandlerThread:16016 [system_info.py:__init__():41] System info init done
2024-03-06 09:46:23,133 INFO    HandlerThread:16016 [system_monitor.py:start():194] Starting system monitor
2024-03-06 09:46:23,133 INFO    SystemMonitor:16016 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-03-06 09:46:23,133 INFO    HandlerThread:16016 [system_monitor.py:probe():214] Collecting system info
2024-03-06 09:46:23,136 INFO    SystemMonitor:16016 [interfaces.py:start():190] Started cpu monitoring
2024-03-06 09:46:23,136 DEBUG   HandlerThread:16016 [system_info.py:probe():150] Probing system
2024-03-06 09:46:23,137 INFO    SystemMonitor:16016 [interfaces.py:start():190] Started disk monitoring
2024-03-06 09:46:23,137 INFO    SystemMonitor:16016 [interfaces.py:start():190] Started gpu monitoring
2024-03-06 09:46:23,138 INFO    SystemMonitor:16016 [interfaces.py:start():190] Started memory monitoring
2024-03-06 09:46:23,146 INFO    SystemMonitor:16016 [interfaces.py:start():190] Started network monitoring
2024-03-06 09:46:23,147 DEBUG   HandlerThread:16016 [system_info.py:_probe_git():135] Probing git
2024-03-06 09:46:23,184 DEBUG   HandlerThread:16016 [system_info.py:_probe_git():143] Probing git done
2024-03-06 09:46:23,184 DEBUG   HandlerThread:16016 [system_info.py:probe():198] Probing system done
2024-03-06 09:46:23,184 DEBUG   HandlerThread:16016 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22000-SP0', 'python': '3.11.5', 'heartbeatAt': '2024-03-06T04:16:23.136437', 'startedAt': '2024-03-06T04:16:13.399550', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'Assignment_2/q2.ipynb', 'codePathLocal': None, 'root': 'C:/Users/hp', 'git': {'remote': None, 'commit': None}, 'email': 'tejas591995@gmail.com', 'host': 'LAPTOP-ACVEONIK', 'username': 'hp', 'executable': 'C:\\Users\\hp\\anaconda3\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 3302.0, 'min': 0.0, 'max': 3302.0}, 'cpu_freq_per_core': [{'current': 3302.0, 'min': 0.0, 'max': 3302.0}], 'disk': {'/': {'total': 176.14648056030273, 'used': 170.22632598876953}}, 'gpu': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'memory_total': 4294967296}], 'memory': {'total': 15.751945495605469}}
2024-03-06 09:46:23,185 INFO    HandlerThread:16016 [system_monitor.py:probe():224] Finished collecting system info
2024-03-06 09:46:23,185 INFO    HandlerThread:16016 [system_monitor.py:probe():227] Publishing system info
2024-03-06 09:46:23,185 DEBUG   HandlerThread:16016 [system_info.py:_save_conda():207] Saving list of conda packages installed into the current environment
2024-03-06 09:46:23,690 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\conda-environment.yaml
2024-03-06 09:46:33,304 DEBUG   HandlerThread:16016 [system_info.py:_save_conda():219] Saving conda packages done
2024-03-06 09:46:33,305 INFO    HandlerThread:16016 [system_monitor.py:probe():229] Finished publishing system info
2024-03-06 09:46:33,305 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:46:33,305 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: keepalive
2024-03-06 09:46:33,305 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:46:33,305 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: keepalive
2024-03-06 09:46:33,306 DEBUG   SenderThread:16016 [sender.py:send():378] send: files
2024-03-06 09:46:33,306 INFO    SenderThread:16016 [sender.py:_save_file():1365] saving file wandb-metadata.json with policy now
2024-03-06 09:46:33,757 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\conda-environment.yaml
2024-03-06 09:46:33,757 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\wandb-metadata.json
2024-03-06 09:46:33,845 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: python_packages
2024-03-06 09:46:33,845 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: python_packages
2024-03-06 09:46:33,846 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: stop_status
2024-03-06 09:46:33,846 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-06 09:46:33,846 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: stop_status
2024-03-06 09:46:34,569 DEBUG   SenderThread:16016 [sender.py:send():378] send: telemetry
2024-03-06 09:46:34,604 INFO    wandb-upload_0:16016 [upload_job.py:push():131] Uploaded file C:\Users\hp\AppData\Local\Temp\tmpehmpi54jwandb\40nvx0u0-wandb-metadata.json
2024-03-06 09:46:34,769 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\requirements.txt
2024-03-06 09:46:36,596 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:46:41,629 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:46:46,664 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:46:47,904 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: partial_history
2024-03-06 09:46:47,905 DEBUG   SenderThread:16016 [sender.py:send():378] send: history
2024-03-06 09:46:47,905 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: summary_record
2024-03-06 09:46:47,906 INFO    SenderThread:16016 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-06 09:46:48,850 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-06 09:46:48,865 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: stop_status
2024-03-06 09:46:48,866 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: stop_status
2024-03-06 09:46:48,896 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\wandb-summary.json
2024-03-06 09:46:48,896 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\output.log
2024-03-06 09:46:50,918 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\output.log
2024-03-06 09:46:52,254 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:46:52,941 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\config.yaml
2024-03-06 09:46:57,584 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:00,322 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: partial_history
2024-03-06 09:47:00,322 DEBUG   SenderThread:16016 [sender.py:send():378] send: history
2024-03-06 09:47:00,323 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: summary_record
2024-03-06 09:47:00,324 INFO    SenderThread:16016 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-06 09:47:01,002 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\wandb-summary.json
2024-03-06 09:47:02,006 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\output.log
2024-03-06 09:47:03,333 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:03,864 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: stop_status
2024-03-06 09:47:03,865 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: stop_status
2024-03-06 09:47:03,867 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-06 09:47:09,190 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:13,304 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: partial_history
2024-03-06 09:47:13,305 DEBUG   SenderThread:16016 [sender.py:send():378] send: history
2024-03-06 09:47:13,305 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: summary_record
2024-03-06 09:47:13,306 INFO    SenderThread:16016 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-06 09:47:14,089 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\wandb-summary.json
2024-03-06 09:47:14,089 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\output.log
2024-03-06 09:47:14,322 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:18,880 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: stop_status
2024-03-06 09:47:18,880 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: stop_status
2024-03-06 09:47:18,882 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-06 09:47:22,724 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:23,161 DEBUG   SystemMonitor:16016 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-03-06 09:47:23,162 DEBUG   SenderThread:16016 [sender.py:send():378] send: stats
2024-03-06 09:47:25,953 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: partial_history
2024-03-06 09:47:25,954 DEBUG   SenderThread:16016 [sender.py:send():378] send: history
2024-03-06 09:47:25,954 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: summary_record
2024-03-06 09:47:25,955 INFO    SenderThread:16016 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-06 09:47:26,203 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\wandb-summary.json
2024-03-06 09:47:27,981 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:28,230 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\output.log
2024-03-06 09:47:33,033 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:33,894 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-06 09:47:33,894 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: stop_status
2024-03-06 09:47:33,894 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: stop_status
2024-03-06 09:47:37,985 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: partial_history
2024-03-06 09:47:37,986 DEBUG   SenderThread:16016 [sender.py:send():378] send: history
2024-03-06 09:47:37,986 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: summary_record
2024-03-06 09:47:37,987 INFO    SenderThread:16016 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-06 09:47:38,328 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\wandb-summary.json
2024-03-06 09:47:38,999 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:40,347 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\output.log
2024-03-06 09:47:44,029 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:48,899 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: stop_status
2024-03-06 09:47:48,900 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-06 09:47:48,900 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: stop_status
2024-03-06 09:47:49,269 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:47:53,182 DEBUG   SenderThread:16016 [sender.py:send():378] send: stats
2024-03-06 09:47:55,205 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:48:00,232 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:48:03,495 ERROR   FileStreamThread:16016 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/tejas591995/mnist-training/oyjusw69/file_stream
NoneType: None
2024-03-06 09:48:03,899 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-06 09:48:03,900 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: stop_status
2024-03-06 09:48:03,900 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: stop_status
2024-03-06 09:48:05,729 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:48:10,773 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:48:15,808 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:48:18,909 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-06 09:48:18,909 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: stop_status
2024-03-06 09:48:18,910 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: stop_status
2024-03-06 09:48:21,250 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:48:23,186 DEBUG   SenderThread:16016 [sender.py:send():378] send: stats
2024-03-06 09:48:23,456 ERROR   FileStreamThread:16016 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/tejas591995/mnist-training/oyjusw69/file_stream
NoneType: None
2024-03-06 09:48:27,213 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:48:32,257 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:48:33,214 DEBUG   SenderThread:16016 [sender.py:send():378] send: telemetry
2024-03-06 09:48:33,214 DEBUG   SenderThread:16016 [sender.py:send():378] send: exit
2024-03-06 09:48:33,214 INFO    SenderThread:16016 [sender.py:send_exit():585] handling exit code: 0
2024-03-06 09:48:33,214 INFO    SenderThread:16016 [sender.py:send_exit():587] handling runtime: 130
2024-03-06 09:48:33,215 INFO    SenderThread:16016 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-06 09:48:33,216 INFO    SenderThread:16016 [sender.py:send_exit():593] send defer
2024-03-06 09:48:33,216 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: defer
2024-03-06 09:48:33,216 INFO    HandlerThread:16016 [handler.py:handle_request_defer():172] handle defer: 0
2024-03-06 09:48:33,216 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: defer
2024-03-06 09:48:33,216 INFO    SenderThread:16016 [sender.py:send_request_defer():609] handle sender defer: 0
2024-03-06 09:48:33,216 INFO    SenderThread:16016 [sender.py:transition_state():613] send defer: 1
2024-03-06 09:48:33,216 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: defer
2024-03-06 09:48:33,216 INFO    HandlerThread:16016 [handler.py:handle_request_defer():172] handle defer: 1
2024-03-06 09:48:33,216 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: defer
2024-03-06 09:48:33,217 INFO    SenderThread:16016 [sender.py:send_request_defer():609] handle sender defer: 1
2024-03-06 09:48:33,217 INFO    SenderThread:16016 [sender.py:transition_state():613] send defer: 2
2024-03-06 09:48:33,217 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: defer
2024-03-06 09:48:33,217 INFO    HandlerThread:16016 [handler.py:handle_request_defer():172] handle defer: 2
2024-03-06 09:48:33,217 INFO    HandlerThread:16016 [system_monitor.py:finish():203] Stopping system monitor
2024-03-06 09:48:33,217 INFO    HandlerThread:16016 [interfaces.py:finish():202] Joined cpu monitor
2024-03-06 09:48:33,217 INFO    HandlerThread:16016 [interfaces.py:finish():202] Joined disk monitor
2024-03-06 09:48:33,218 DEBUG   SystemMonitor:16016 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-03-06 09:48:33,226 DEBUG   SystemMonitor:16016 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-03-06 09:48:33,251 INFO    HandlerThread:16016 [interfaces.py:finish():202] Joined gpu monitor
2024-03-06 09:48:33,251 INFO    HandlerThread:16016 [interfaces.py:finish():202] Joined memory monitor
2024-03-06 09:48:33,251 INFO    HandlerThread:16016 [interfaces.py:finish():202] Joined network monitor
2024-03-06 09:48:33,252 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: defer
2024-03-06 09:48:33,252 INFO    SenderThread:16016 [sender.py:send_request_defer():609] handle sender defer: 2
2024-03-06 09:48:33,252 INFO    SenderThread:16016 [sender.py:transition_state():613] send defer: 3
2024-03-06 09:48:33,252 DEBUG   SenderThread:16016 [sender.py:send():378] send: stats
2024-03-06 09:48:33,252 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: defer
2024-03-06 09:48:33,252 INFO    HandlerThread:16016 [handler.py:handle_request_defer():172] handle defer: 3
2024-03-06 09:48:33,252 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: defer
2024-03-06 09:48:33,252 INFO    SenderThread:16016 [sender.py:send_request_defer():609] handle sender defer: 3
2024-03-06 09:48:33,252 INFO    SenderThread:16016 [sender.py:transition_state():613] send defer: 4
2024-03-06 09:48:33,253 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: defer
2024-03-06 09:48:33,253 INFO    HandlerThread:16016 [handler.py:handle_request_defer():172] handle defer: 4
2024-03-06 09:48:33,253 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: defer
2024-03-06 09:48:33,253 INFO    SenderThread:16016 [sender.py:send_request_defer():609] handle sender defer: 4
2024-03-06 09:48:33,253 INFO    SenderThread:16016 [sender.py:transition_state():613] send defer: 5
2024-03-06 09:48:33,253 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: defer
2024-03-06 09:48:33,253 INFO    HandlerThread:16016 [handler.py:handle_request_defer():172] handle defer: 5
2024-03-06 09:48:33,254 DEBUG   SenderThread:16016 [sender.py:send():378] send: summary
2024-03-06 09:48:33,254 INFO    SenderThread:16016 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-06 09:48:33,254 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: defer
2024-03-06 09:48:33,254 INFO    SenderThread:16016 [sender.py:send_request_defer():609] handle sender defer: 5
2024-03-06 09:48:33,254 INFO    SenderThread:16016 [sender.py:transition_state():613] send defer: 6
2024-03-06 09:48:33,254 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: defer
2024-03-06 09:48:33,254 INFO    HandlerThread:16016 [handler.py:handle_request_defer():172] handle defer: 6
2024-03-06 09:48:33,255 DEBUG   SenderThread:16016 [sender.py:send_request():405] send_request: defer
2024-03-06 09:48:33,255 INFO    SenderThread:16016 [sender.py:send_request_defer():609] handle sender defer: 6
2024-03-06 09:48:33,256 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: status_report
2024-03-06 09:48:33,481 ERROR   FileStreamThread:16016 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/tejas591995/mnist-training/oyjusw69/file_stream
NoneType: None
2024-03-06 09:48:33,547 ERROR   SenderThread:16016 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-03-06 09:48:33,547 ERROR   SenderThread:16016 [internal_api.py:execute():374] {"errors":[{"message":"run tejas591995/mnist-training/oyjusw69 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-03-06 09:48:33,768 INFO    Thread-31 :16016 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\files\wandb-summary.json
2024-03-06 09:48:34,216 DEBUG   HandlerThread:16016 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-06 09:48:35,065 ERROR   SenderThread:16016 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-03-06 09:48:35,065 ERROR   SenderThread:16016 [internal_api.py:execute():374] {"errors":[{"message":"run tejas591995/mnist-training/oyjusw69 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-03-06 09:48:37,532 ERROR   SenderThread:16016 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-03-06 09:48:37,532 ERROR   SenderThread:16016 [internal_api.py:execute():374] {"errors":[{"message":"run tejas591995/mnist-training/oyjusw69 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-03-06 09:48:37,532 INFO    SenderThread:16016 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\lib\retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\internal_api.py", line 369, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\lib\gql_request.py", line 59, in execute
    request.raise_for_status()
  File "C:\Users\hp\anaconda3\Lib\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql
2024-03-06 09:48:42,037 ERROR   SenderThread:16016 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-03-06 09:48:42,037 ERROR   SenderThread:16016 [internal_api.py:execute():374] {"errors":[{"message":"run tejas591995/mnist-training/oyjusw69 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-03-06 09:48:43,753 ERROR   FileStreamThread:16016 [file_stream.py:_handle_response():504] dropped chunk 404 Client Error: Not Found for url: https://api.wandb.ai/files/tejas591995/mnist-training/oyjusw69/file_stream
NoneType: None
2024-03-06 09:48:52,822 ERROR   SenderThread:16016 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-03-06 09:48:52,822 ERROR   SenderThread:16016 [internal_api.py:execute():374] {"errors":[{"message":"run tejas591995/mnist-training/oyjusw69 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-03-06 09:49:11,117 ERROR   SenderThread:16016 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-03-06 09:49:11,117 ERROR   SenderThread:16016 [internal_api.py:execute():374] {"errors":[{"message":"run tejas591995/mnist-training/oyjusw69 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-03-06 09:49:47,342 ERROR   SenderThread:16016 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-03-06 09:49:47,342 ERROR   SenderThread:16016 [internal_api.py:execute():374] {"errors":[{"message":"run tejas591995/mnist-training/oyjusw69 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-03-06 09:50:59,684 ERROR   SenderThread:16016 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-03-06 09:50:59,684 ERROR   SenderThread:16016 [internal_api.py:execute():374] {"errors":[{"message":"run tejas591995/mnist-training/oyjusw69 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-03-06 09:51:00,392 INFO    HandlerThread:16016 [handler.py:finish():869] shutting down handler
2024-03-06 09:51:00,392 INFO    WriterThread:16016 [datastore.py:close():296] close: C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\run-oyjusw69.wandb
2024-03-06 09:51:00,393 ERROR   StreamThr :16016 [internal.py:wandb_internal():174] Thread SenderThread:
Traceback (most recent call last):
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\apis\normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\internal_api.py", line 2216, in upsert_run
    response = self.gql(
               ^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\internal_api.py", line 341, in gql
    ret = self._retry_gql(
          ^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\lib\retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\internal_api.py", line 369, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\lib\gql_request.py", line 59, in execute
    request.raise_for_status()
  File "C:\Users\hp\anaconda3\Lib\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\internal_util.py", line 49, in run
    self._run()
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\internal_util.py", line 100, in _run
    self._process(record)
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\internal.py", line 328, in _process
    self._sm.send(record)
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\sender.py", line 385, in send
    send_handler(record)
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\sender.py", line 407, in send_request
    send_handler(record)
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\sender.py", line 635, in send_request_defer
    self.debounce(final=True)
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\sender.py", line 536, in debounce
    self._maybe_update_config(always=final)
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\sender.py", line 513, in _maybe_update_config
    self._debounce_config()
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\sdk\internal\sender.py", line 542, in _debounce_config
    self._api.upsert_run(
  File "C:\Users\hp\anaconda3\Lib\site-packages\wandb\apis\normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: run tejas591995/mnist-training/oyjusw69 was previously created and deleted; try a new run name (Error 409: Conflict)
